lib/ansible/modules/cloud/amazon/rds_option_group.py

```diff
@@ -0,0 +1,384 @@
+                            # Security groups need to be handled separately due to different keys on request and what is
+                            # returned by the API
+                            if name in option.iterkeys() and name != 'OptionSettings' and name != 'VpcSecurityGroupMemberships':
+                                if cmp(setting_name[name], option[name]) != 0:
+                                    requires_update = True
+                            if name in option.iterkeys() and name == 'VpcSecurityGroupMemberships':
+                                for groups in option[name]:
+                                    if groups['VpcSecurityGroupId'] not in setting_name[name]:
+                                        requires_update = True
+
+                    for new_option in new_options:
+                        if option['OptionName'] == new_option['OptionName']:
+                            for current_option_setting in option['OptionSettings']:
+                                for new_option_setting in new_option['OptionSettings']:
+                                    if new_option_setting['Name'] == current_option_setting['Name']:
+                                        if cmp(new_option_setting['Value'], current_option_setting['Value']) != 0:
+                                            requires_update = True
+
+    return requires_update
+
+
+def compare_option_group(client, module):
+    current_option_group = get_option_group(client, module)
+    new_options = module.params.get('options')
+    to_be_added = None
+    to_be_removed = None
+    for current_option in current_option_group['OptionGroupsList']:
+        # Catch new settings you've provided that aren't in the current settings
+        old_settings = []
+        new_settings = []
+        for setting_name in new_options:
+            new_settings.append(setting_name['OptionName'])
+        for option in current_option['Options']:
+            old_settings.append(option['OptionName'])
+        if cmp(sorted(old_settings),sorted(new_settings)) != 0:
+            to_be_added = list(set(new_settings) - set(old_settings))
+            to_be_removed = list(set(old_settings) - set(new_settings))
+        return to_be_added, to_be_removed
+
+
+def setup_rds_option_group(client, module):
+    results = []
+    changed = False
+
+    # Check if there is an existing options group
+    existing_option_group = get_option_group(client, module)
+
+    if existing_option_group:
+        results = existing_option_group
+        if module.params.get('options'):
+            # Check if existing options require updating
+            update_required = match_option_group_options(client, module)
+            # Check if there are options to be added or removed
+            to_be_added, to_be_removed = compare_option_group(client, module)
+            if to_be_added or update_required:
+                changed, new_option_group_options =  create_option_group_options(client, module)
```

Comment:
You inadvertently added a third blank line. Probably worth running flake8 and removing what issues you can